src/astro/files/base.py

```diff
@@ -36,11 +36,11 @@ def __init__(
 
     @property
     def path(self) -> str:
-        return self.location.path
+        return self.location.path  # type: ignore
```

Comment:
We should solve this instead of ignoring it -- can be in a separate PR

Reply:
@kaxil I'll raise a separate PR for this.